DscResources/AccessControlResourceHelper/AccessControlResourceHelper.psm1

```diff
@@ -168,11 +176,11 @@ Function Get-SchemaObjectName
 
     If($SchemaIdGuid)
     {
```

Comment:
This won't find the rights guid name.  It should be a reverse lookup version of the function update you made to Get-DelegationRightsGuid